### PR TITLE
Added support for camera statuses check

### LINKED ISSUE #59: emergency brake for network disconnecting for robots

### PR DESCRIPTION
**Pull request by anastasiia-kornilova**
Fix #59

### CODE REVIEW

File: LARCmaCS/receiverWorker.cpp [hunk 1]
```diff
@@ -25,11 +25,18 @@ const QString ReceiverWorker::visionIP = QStringLiteral("224.5.23.2");
...
 {
+	mCameraStatuses.resize(Constants::numOfCameras);
+	for (int i = 0; i < Constants::numOfCameras; i++) {
+		mCameraStatuses[i] = false;
+	}
 	mStatisticsTimer.setInterval(1000);
+	mCameraCheckTimer.setInterval(1000);
 
 	connect(&mStatisticsTimer, SIGNAL(timeout()), this, SLOT(formStatistics()));
+	connect(&mStatisticsTimer, SIGNAL(timeout()), this, SLOT(updateCameraStatus()));
```
Comment (DmiitriyJarosh) [inline LARCmaCS/receiverWorker.cpp]: we have another timer for this slot

File: LARCmaCS/receiverWorker.cpp [hunk 2]
```diff
@@ -102,6 +112,16 @@ void ReceiverWorker::processPendingDatagrams()
 	}
 }
 
+void ReceiverWorker::updateCameraStatus()
```
Comment (DmiitriyJarosh) [inline LARCmaCS/receiverWorker.cpp]: What about some info to GUI?

Comment (anastasiia-kornilova) [reply LARCmaCS/receiverWorker.cpp]: Let's add this idea to GUI issues

**Comment by anastasiia-kornilova** [comment]
@DmiitriyJarosh , what about review?

**Pull request merged**